libfreerdp/cache/glyph.c

```diff
@@ -382,6 +392,11 @@ void update_gdi_cache_glyph_v2(rdpContext* context, CACHE_GLYPH_V2_ORDER* cacheG
 		glyphData = &cacheGlyphV2->glyphData[i];
 
 		glyph = Glyph_Alloc(context);
+		if (!glyph)
+		{
+			/* TODO: cleanup perviosly allocated glyph memory in error case */
```

Comment:
@hardening possibly not but I didn't look into the details that's why I added the TODO.